src/plugins/unpacking/hp/internal/pjl_helper.py

```diff
@@ -0,0 +1,102 @@
+import re
+from pathlib import Path
+
+from common_helper_files import get_safe_name, write_binary_to_file
+
+NAME_FIELD_MAX = 102
+
+
+def get_pjl_commands(input_data):
+    pjl_instructions = []
+    tmp = list(re.finditer(b'@PJL [ =.!,?\w]+', input_data))
```

Comment:
`DeprecationWarning: invalid escape sequence \w`
should probably be a raw string `rb'...'`